android/src/main/java/com/reactnative/googlecast/api/RNGCRemoteMediaClient.java

```diff
@@ -99,6 +99,26 @@ public PendingResult execute(RemoteMediaClient client) {
     }, promise);
   }
 
+  @ReactMethod
+  public void queueNext(@Nullable final ReadableMap customData, final Promise promise) {
+    with.withX(new With.WithXPromisify<RemoteMediaClient>() {
+      @Override
+      public PendingResult execute(RemoteMediaClient client) {
+        return client.queueNext(RNGCJSONObject.fromJson(customData));
+      }
+    }, promise);
+  }
+
+  @ReactMethod
+  public void queuePrev(@Nullable final ReadableMap customData, final Promise promise) {
+    with.withX(new With.WithXPromisify<RemoteMediaClient>() {
+      @Override
+      public PendingResult execute(RemoteMediaClient client) {
+        return client.queuePrev(RNGCJSONObject.fromJson(customData));
+      }
+    }, promise);
+  }
```

Comment:
Could you move these after the `queueInsert...` methods so that they're sorted alphabetically? (Also in the other files.)